Modules/timemodule.c

```diff
@@ -2213,7 +2214,10 @@ static int
 pysleep(PyTime_t timeout)
 {
     assert(timeout >= 0);
-
+    assert(!PyErr_Occurred());
+    if (timeout == 0) { // gh-125997
+        return pysleep_zero();
+    }
```

Comment:
Empty line for readability:

```suggestion
    }

```